components/org.wso2.carbon.identity.oauth.stub/src/main/resources/OAuthAdminService.wsdl

```diff
@@ -394,6 +394,7 @@
             <xs:complexType name="OAuthConsumerAppDTO">
                 <xs:sequence>
                     <xs:element minOccurs="0" name="OAuthVersion" nillable="true" type="xs:string"/>
+                    <xs:element maxOccurs="unbounded" minOccurs="0" name="accessTokenAudiences" nillable="true" type="xs:string"/>
```

Comment:
we will need to apply this change for the application management REST API as well